assets/js/frontend-variation-cart.js

```diff
@@ -1,5 +1,16 @@
 jQuery(document).ready(function($){
 
+
+	//Alter scroll nature
+$.scroll_to_notices=function( scrollElement ) {

```

Comment:
@Preciousomonze what is the change to behavior here? How is that going to effect anyone else using `$.scroll_to_notices`?

Reply:
Might edit it, still not working, just wanted to push my commits :)

Reply:
trying as much not to alter it for others, i was trying to test out something.